scripts/zulip_emoji_merge_delegate.py

```diff
@@ -20,60 +20,61 @@
     site=ZULIP_SITE
 )
 
-# Fetch the last 200 messages
-response = client.get_messages({
-    "anchor": "newest",
-    "num_before": 200,
-    "num_after": 0,
-    "narrow": [{"operator": "channel", "operand": "PR reviews"}],
-})
-
-messages = response['messages']
-pr_pattern = re.compile(r'https://github\.com/leanprover-community/mathlib4/pull/(\d+)')
-
-for message in messages:
-    content = message['content']
-    match = pr_pattern.search(content)
-    if match:
-        pr_number = match.group(1)
-        # Check for emoji reactions
-        reactions = message['reactions']
-        has_peace_sign = any(reaction['emoji_name'] == 'peace_sign' for reaction in reactions)
-        has_bors = any(reaction['emoji_name'] == 'bors' for reaction in reactions)
-        has_merge = any(reaction['emoji_name'] == 'merge' for reaction in reactions)
-
-        pr_url = f"https://api.github.com/repos/leanprover-community/mathlib4/pulls/{pr_number}"
-        pr_response = requests.get(pr_url, headers={"Authorization": GITHUB_TOKEN})
-        pr_data = pr_response.json()
-        labels = [label['name'] for label in pr_data['labels']]
-
-        if 'delegated' in labels:
-            client.add_reaction({
-                "message_id": message['id'],
-                "emoji_name": "peace_sign"
-            })
-        elif 'ready-to-merge' in labels:
-            if has_peace_sign:
-                client.remove_reaction({
+for channel in ["PR reviews", "mathlib reviewers"]:
+    # Fetch the last 1000 messages
+    response = client.get_messages({
+        "anchor": "newest",
+        "num_before": 1000,
+        "num_after": 0,
+        "narrow": [{"operator": "channel", "operand": channel}],
+    })
+    
+    messages = response['messages']
+    pr_pattern = re.compile(r'https://github\.com/leanprover-community/mathlib4/pull/(\d+)')
+    
+    for message in messages:
+        content = message['content']
+        match = pr_pattern.search(content)
+        if match:
+            pr_number = match.group(1)
+            # Check for emoji reactions
+            reactions = message['reactions']
+            has_peace_sign = any(reaction['emoji_name'] == 'peace_sign' for reaction in reactions)
+            has_bors = any(reaction['emoji_name'] == 'bors' for reaction in reactions)
+            has_merge = any(reaction['emoji_name'] == 'merge' for reaction in reactions)
+    
+            pr_url = f"https://api.github.com/repos/leanprover-community/mathlib4/pulls/{pr_number}"
+            pr_response = requests.get(pr_url, headers={"Authorization": GITHUB_TOKEN})
+            pr_data = pr_response.json()
+            labels = [label['name'] for label in pr_data['labels']]
+    
```

Comment:
(You have some trailing whitespace, e.g. on this line. I have configured my VS Code to remove it automatically. Not worth a big fuss, but perhaps helpful for you to know.)

Reply:
Using `git show` locally told me.